cmd/janus/init.go

```diff
@@ -64,19 +65,21 @@ func init() {
 	log.Debugf("Trying to connect to redis pool: %s", dsn)
 	storage, err = store.NewRedisStore(pool)
 	if err != nil {
-		log.Fatalf("Couldn't connect to the redis pool: %s", err.Error())
+		log.WithError(err).Fatal("Couldn't connect to the redis pool")
 	}
 }
 
 // initializes new StatsD client if it enabled
 func init() {
 	var options []statsd.Option
 	statsdConfig := globalConfig.Statsd
+	muted := false
 
 	log.Debugf("Trying to connect to statsd instance: %s", statsdConfig.DSN)
 	if len(statsdConfig.DSN) == 0 {
 		log.Debug("Statsd DSN not provided, client will be muted")
 		options = append(options, statsd.Mute(true))
+		muted = true
```

Comment:
If you do `statsd.Mute` you already have the value... I think the `muted` extra var is unecessary